docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,26 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |add details to a certain contact after adding them |have more information available afterwards
+
+|`* * *` |user |be notified if I were to delete a contact |I don’t mistakenly delete a contact
```

Comment:
This is more like a detail that should be recorded in the use case of `delete contact`'s user story.